pymc4/distributions/tensorflow/continuous.py

```diff
@@ -1,6 +1,4 @@
-"""
-PyMC4 continuous random variables for tensorflow.
-"""
+"""PyMC4 continuous random variables for tensorflow."""
 import tensorflow_probability as tfp
 from pymc4.distributions import abstract
```

Comment:
being picky here, should we use relative imports for inside the library? Not part of this PR but just wanted to ask

Reply:
I use doctest to test code snippets in documentation. Doctest is complaining if import is relative sometimes

Reply:
I had that problem with pytest, too -- complaints about relative imports -- and it turned out for me it was because I was running the tests (this is for PyMC3) *inside* the source directory.  When I ran it "above" the directory (i.e., from `pymc3` instead of `pymc3/pymc3/`) the complaints about relative imports went away.

I think the advantage of relative imports is that they don't risk an import cycle as much as absolute ones do.

Reply:
> I think the advantage of relative imports is that they don't risk an import cycle as much as absolute ones do.

I believe this part is true, that relative imports risk less that another package of same name will be imported from `sys.path` versus the adjacent module

Reply:
https://stackoverflow.com/a/4209771

Reply:
My personal feeling is that it is hard to run into a problem you describe, you would not import pymc4 in the first place. But for our use case we can test code snippets in docs without mess

Reply:
No worries, for this PR just ignore me there's more important things :) thanks @ferrine